python-frontend/src/test/java/org/sonar/python/semantic/MatchStatementSymbolsTest.java

```diff
@@ -0,0 +1,177 @@
+/*
```

Comment:
For the sake of completeness, maybe we could have one test case with using guards as well?

Reply:
Good point, thanks!